content/algorithms/expected_consensus/_index.md

```diff
@@ -401,6 +401,6 @@ A single consensus fault results into:
 
 ### Detection and Reporting
 
-A node that detects and report a consensus fault is called "slasher", any user in Filecoin can be a slasher. They can report consensus faults by calling the `ReportConsensusFault` on the `StorageMinerActor` of the faulty miner. The slasher is rewarded with a portion of the offending miner's [Pledge Collateral](miner_collaterals) for notifying the network of the consensu fault.
+A node that detects and reports a consensus fault is called "slasher". Any user in Filecoin can be a slasher. They can report consensus faults by calling the `ReportConsensusFault` on the `StorageMinerActor` of the faulty miner. The slasher is rewarded with a portion of the _offending miner's block reward_ (as opposed to the miner's initial pledge) for notifying the network of the consensus fault. The reward should not be taken from the offending miner's initial pledge as the miner could have spent their initial pledge in earlier penalties and therefore cause more harm to the network. Recall that a miner who causes a consensus fault on purpose is the first to know about it (i.e., the network can only discover later) and can report it first.
```

Comment:
I checked the code, I suggest to write this 

"A node that detects and reports a consensus fault is called "slasher". Any user in Filecoin can be a slasher. They can report consensus faults by calling the `ReportConsensusFault` on the `StorageMinerActor` of the faulty miner. The slasher is rewarded with a portion of the penalty paid by the offending miner `ConsensusFaultPenalty` for notifying the network of the consensus fault. Note that some slasher might not get the full reward because of a low balance for the offending miners, however for honest miner is still rational to notify the network about consensus faults."

And remove the rest:

~The reward should not be taken from the offending miner's initial pledge as the miner could have spent their initial pledge in earlier penalties and therefore cause more harm to the network. Recall that a miner who causes a consensus fault on purpose is the first to know about it (i.e., the network can only discover later) and can report it first.~

Reply:
fine with me!